v0/destinations/salesforce/transform.js

```diff
@@ -45,6 +46,11 @@ async function getSFDCHeader(destination) {
     response = await fetchWithProxy(authUrl, { method: "POST", body: {} });
     response = await response.json();
   } catch (error) {
+    const splitError = error.message.split("&");
+    const email = splitError[0].substring(splitError[0].indexOf("=") + 1);
+    const password = splitError[1].substring(splitError[1].indexOf("=") + 1);
+    error.message = error.message.replace(email, SHA256(email).toString());
+    error.message = error.message.replace(password, SHA256(password).toString());
```

Comment:
We don't need SHA encode it just mask it to ****

Reply:
I think you are looking over outdated code. SHA is not used.